### DISCUSSION TITLE
Calibration tab not showing up

### DISCUSSION THREAD

**Discussion by umpmanjoe** [OrcaSlicer/OrcaSlicer#2918] [Q&A]
I have Orca Slicer installed and I have 3 printers connected to it.  My Bambu X1C, my Voron 2.4 and a Qidi X Max 3.   The calibration tab only shows up when I use the bambu printer

**Comment by LJKraus** [comment]
The calibration _tab_ is inherited from Bambu Studio and only works with Bambu printers.  It is very limited and stores the result on their printers. The results are lost if you ever reset the firmware.

The OrcaSlicer calibrations are available on the top line and should work with all printers. You store your results in the OrcaSlicer filament profiles, where they can be included in your computer backups.

![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/80ae8e33-b3d1-4707-aa67-35f97dfdf80b)
